dir.targets

```diff
@@ -55,8 +56,13 @@
       <RuntimePath>$(RuntimePath)</RuntimePath>
     </BinPlaceConfiguration>
 
+    <BinPlaceConfiguration Condition="'$(IsNETCoreApp)' == 'true' AND '$(IsTestProject)' == 'true' AND '$(BinPlaceNETCoreAppTests)' == 'true'" Include="netcoreapp-$(_bc_OSGroup)">
+      <PackageFileRuntimePath>$(NETCoreAppPackageRuntimePath)\..\tests</PackageFileRuntimePath>
```

Comment:
You don't need PackageFileRuntimePath if all you want to do is get the files copied into the tests folder.